.github/workflows/rust.yml

```diff
@@ -30,10 +30,14 @@ jobs:
       run: cargo fmt --verbose --all -- --check
     - name: Clippy check
       run: cargo clippy --verbose --all-targets
+    - name: Clippy check with all features
+      run: cargo clippy --verbose --all-targets --all-features
     - name: Cargo check without features
       run: cargo check --all-targets --manifest-path "scylla/Cargo.toml" --features ""
     - name: Cargo check with all serialization features
       run: cargo check --all-targets --manifest-path "scylla/Cargo.toml" --features "full-serialization"
+    - name: Cargo check with all features
+      run: cargo check --all-targets --manifest-path "scylla/Cargo.toml" --all-features
```

Comment:
Not related to this PR: maybe we should also check `scylla-cql` separately with all features and with no features?
I can remember that at one moment I nearly rendered `scylla-cql` not compilable with no features, but with the workspace-level `cargo check` one couldn't experienced it, because `scylla` unconditionally enabled `chrono` dependency in `scylla-cql` IIRC.

Reply:
Imho the `build` CI job should be split into static analysis and tests.
Then in static analysis one we could easily run a lot of combinations, because it should still take less time than tests

Reply:
Opened https://github.com/scylladb/scylla-rust-driver/issues/982